src/core/dom/video/pause-helper.js

```diff
@@ -14,14 +14,33 @@
  * limitations under the License.
  */
 
-import {observeBorderBoxSize, unobserveBorderBoxSize} from './size-observer';
+import {observeBorderBoxSize, unobserveBorderBoxSize} from '../size-observer';
+
+// typedef imports
+import {PausableInterface} from './pausable-interface';
+
+/**
+ * TODO(rcebulko, #34096): Remove this once a proper AMPElement type is
+ * available, and declare that it @implements PausableInterface. For now, this
+ * provides the necessary type strictness.
+ * @constructor
+ * @extends Element
+ * @implements {PausableInterface}
+ */
+export function PausableElement() {}
+/** @function */
+PausableElement.prototype.pause = () => {};
```

Comment:
Use class syntax to keep this declarative.

Reply:
Done